src/sass/user-profile.scss

```diff
@@ -55,4 +55,52 @@
       margin-right: 5px;
     }
   }
+  
+  .saved-form-information {
+    margin: 0;
+    
+    div {
+      margin-bottom: 8px;
+    }
+    
+    span.saved-form-item-metadata {
```

Comment:
Probably don't need the `span` part of this.

Reply:
Specificity is being used here to override other styles.